src/sections/header.html

```diff
@@ -1,11 +1,12 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>header</title>
-</head>
-<body class="font-outfit">
-    <h1 class="text-blue-500">Header</h1>
-</body>
-</html>
\ No newline at end of file
+<nav class="flex justify-between items-center px-10 py-3">
+    <p class="font-semibold">DE<br>EN</p>
+    <img src="/src/img/Baumeister.png" alt="">
```

Comment:
Consider not leaving alt attribute empty, this is the text displayed in case an image were not able to be displayed and it is also need for screen readers